modules/stochastic_tools/doc/content/syntax/Trainers/index.md

```diff
@@ -14,6 +14,8 @@ through the rows of a given [sampler](Samplers/index.md), specified by the
 
 !listing SurrogateTrainer.C re=void\sSurrogateTrainer::execute.*?^}
 
+!listing SurrogateTrainer.C re=void\sSurrogateTrainer::executeTraining.*?^}
```

Comment:
I added this listing since we separated out the training loop (contents of `executeTraining()`) from `execute()`.